templates/advanced/actions.py

```diff
@@ -33,20 +30,23 @@ def get_repository_commits(
     Returns:
         A list of commit messages.
     """
+    from sema4ai.action_server._session import session
```

Comment:
I don't think this can be done as is... the `templates` can't really import the `action_server`. If we need the user to use rcc settings in actions we'd need a different design.

Wouldn't it be enough for actions to have the env variables set by the action server?

Reply:
Thanks for noticing this 👍 I didn't realize this was another context.

Yes we can just set the environment variables as we are. Unfortunately the SSL verify option doesn't seem to work with just the environment variable. Also the allow legacy renegotiation doesn't have any environment variable.